README.md

```diff
@@ -33,6 +33,85 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Getting started
+
+### Prerequisites
+`runc` is expected to be installed on the system. It is picked up as the default runtime by ocid.
```

Comment:
The hard-coded runc path is actually incompatible with current runc Makefile ( /usr/bin/runc instead of /usr/local/sbin/runc ). I would suggest we just dont hardcode the default path.